src/seeds.proposals.cpp

```diff
@@ -524,26 +533,30 @@ void proposals::evalproposal (uint64_t proposal_id, uint64_t prop_cycle) {
           }
         }
 
-        // TODO: if we allow num_cycles == 1, this needs to go into passed instead of evaluate.
-        // uint64_t num_cycles = pitr -> pay_percentages.size() - 1;
+        uint64_t num_cycles = pitr->pay_percentages.size() - 1;
 
         props.modify(pitr, _self, [&](auto & proposal){
           proposal.passed_cycle = prop_cycle;
           proposal.age = 0;
           proposal.staked = asset(0, seeds_symbol);
-          proposal.status = status_evaluate;
+          if (proposal.age >= num_cycles && !is_alliance_type) {
```

Comment:
I don't understand this - why !alliance_type... proposal.age == 0 it was just set above...